File: voice-telephony/call-log/sync-company-call-log/code-samples/syncAccountCallLog.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';

// OPTIONAL QUERY PARAMETERS
$queryParams = array(
    //'syncType' => 'FSync',
    //'syncToken' => '<ENTER VALUE>',
    //'dateFrom' => '<ENTER VALUE>',
    //'recordCount' => 000,
    //'statusGroup' => 'Missed',
    //'view' => 'Simple',
    //'showDeleted' => true
)

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->get("/restapi/v1.0/account/{$accountId}/call-log-sync", $queryParams);
// PROCESS RESPONSE
?>

File: voice-telephony/call-log/sync-company-call-log/code-samples/syncAccountCallLog.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
accountId = '<ENTER VALUE>'

# OPTIONAL QUERY PARAMETERS
queryParams = {
    #'syncType': 'FSync',
    #'syncToken': '<ENTER VALUE>',
    #'dateFrom': '<ENTER VALUE>',
    #'recordCount': 000,
    #'statusGroup': 'Missed',
    #'view': 'Simple',
    #'showDeleted': true
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.get(f'/restapi/v1.0/account/{accountId}/call-log-sync', queryParams)
# PROCESS RESPONSE